src/main/flight/gps_rescue.c

```diff
@@ -953,12 +939,13 @@ float gpsRescueGetImuYawCogGain(void)
 
 float gpsRescueGetThrottle(void)
 {
-    // Calculated a desired commanded throttle scaled from 0.0 to 1.0 for use in the mixer.
-    // We need to compensate for min_check since the throttle value set by gps rescue
-    // is based on the raw rcCommand value commanded by the pilot.
+    // Calculate the commanded throttle scaled from 0.0 to 1.0 for use in the mixer.
```

Comment:
```suggestion
    // Calculate the commanded throttle scaled to 0.0 to 1.0 for use in the mixer.
```

Reply:
used `Calculate the commanded throttle for use in the mixer. Output must be within range 0.0 - 1.0.`